lang/en.json

```diff
@@ -900,18 +900,17 @@
         "SiteWithoutDataReactFollowStepCompleted": "%1$sCongratulations!%2$s You have successfully installed the Matomo Analytics tracking code via the Matomo Tag Manager. To verify that hits are being tracked, visit your website and check that this data is visible in your Matomo instance.",
         "SPAFollowStep1": "Select the right container for your website.%1$sNote:%2$s Matomo automatically pre-configures a container when you add a site. By default, this container is selected in the fields below. If you are planning to use a single container for multiple websites, make sure to select that specific container. You can %3$smanage your containers here%4$s.",
         "SPAFollowStep2": "Navigate to %1$sMatomo Tag Manager's Triggers%2$s.",
-        "SPAFollowStep3": "Check that a trigger called %1$sPageview%2$s already exists. If you don’t see the Pageview trigger, follow %3$sthis guide%4$s.",
-        "SPAFollowStep4": "Click \"Create New Trigger\".",
-        "SPAFollowStep5": "Click %1$sHistory Change%2$s in the User Engagement section.",
+        "SPAFollowStep3": "Check that a trigger of the type %1$s%5$s%2$s already exists. If you don’t see the %5$s trigger, follow %3$sthis guide%4$s.",
+        "SPAFollowStep5": "Click %1$s%3$s%2$s in the %4$s section.",
         "SPAFollowStep7": "Navigate to %1$sMatomo Tag Manager's Tags%2$s.",
-        "SPAFollowStep8": "For the %1$sPageview%2$s tag with the type %1$sMatomo Analytics%2$s, click on the Edit tag icon. If you don’t see the Pageview tag, follow %3$sthis guide%4$s.",
-        "SPAFollowStep9": "In the section \"Configure what this tag should do\", set the Custom Title to %1$s{{PageTitle}}",
-        "SPAFollowStep10": "Set the Custom URL to %1$s{{PageOrigin}}/{{PageHash}}%2$s if your SPA or PWA has a %1$s#%2$s in the URL to navigate to different pages. Otherwise, set it as %1$s{{PageUrl}}%2$s",
-        "SPAFollowStep11": "In the section “Configure when the tag should do this”, under \"Execute this tag when any of these triggers are triggered\", directly under the pre-selected %1$sPageview%2$s trigger, add the %1$sHistory Change%2$s trigger.",
-        "SPAFollowStep12": "Click \"Update\".",
-        "SPAFollowStep13": "In the left menu, click \"Publish\".",
-        "SPAFollowStep14": "Add a version name, for example %1$s1.0%2$s",
-        "SPAFollowStep15": "Click the green \"Create New Version And Publish Release\" button.",
+        "SPAFollowStep8": "For the %1$s%5$s%2$s tag with the type %1$s%6$s%2$s, click on the Edit tag icon. If you don’t see the %5$s tag, follow %3$sthis guide%4$s.",
```

Comment:
```suggestion
        "SPAFollowStep8": "For the %1$s%5$s%2$s tag with the type %1$s%6$s%2$s, click on the Edit tag icon. If you don't see the %5$s tag, follow %3$sthis guide%4$s.",
```

Reply:
I'll change it for now but it's basically the wrong type of punctuation mark we use in these places. It's an opening single quote `'` whereas it should be an apostrophe `’`. A common mistake made in technical texts.

https://practicaltypography.com/apostrophes.html